docs/compose/Calendar2/README.md

```diff
@@ -85,7 +85,7 @@ controller
   .map { it.selection }
   .onEach { selection ->
     when (selection) {
-      is CalendarSelection.None -> showToast("No date is selected")
+      is CalendarSelection.None -> Unit // do nothing
```

Comment:
just {} might be nicer (same for all the other ones in the sample)

Reply:
Done